packages/lesswrong/components/editor/RateLimitWarning.tsx

```diff
@@ -32,15 +53,22 @@ const RateLimitWarning = ({lastRateLimitExpiry, rateLimitMessage}: {
     return `${diffInWeeks} week${diffInWeeks > 1 ? 's' : ''}`
   }
 
-  let message = `Please wait ${getTimeUntilNextPost()} before posting again. ${rateLimitMessage ?? ''}`
+  let message = `<p>You can post again in ${getTimeUntilNextPost()}.</p> ${rateLimitMessage ?? ''}`
```

Comment:
Probably this should mention that you can still save drafts (& we should confirm that users can, in fact, save drafts while rate-limited).